src/web/pages/reportconfigs/dialog.jsx

```diff
@@ -68,6 +68,8 @@ const Param = ({
     field = (
       <YesNoRadio
         convert={parseBoolean}
+        yesValue={true}
+        noValue={false}
```

Comment:
This does not seem to work for me as expected. The default value is not selected when opening the dialog for the first time and trying to set a value does not seem to work (No is always selected). Setting `yesValue={1} noValue={0}` and using the default convert seems to work for me. But re-opening the dialog after saving the values did not show the saved value.

Reply:
a32a6b0 should fix this